extensions/packageManager/src/node/index.ts

```diff
@@ -38,11 +43,85 @@ const normalizeFeed = (feed) => {
         source: 'nuget',
       };
     });
+  } else if (feed.resources) {
+    // this is actually a myget feed that points to the feed we want...
+    const queryEndpoint = feed.resources.find((resource) => resource['@type'] === 'SearchQueryService');
+    if (queryEndpoint) {
+      const raw = await axios.get(queryEndpoint['@id']);
+      return normalizeFeed(raw.data);
+    } else {
+      return [];
+    }
   } else {
+    console.error('Unknown feed format!', feed);
     return null;
   }
 };
 
+
+const getPackageInfo = async (name, url) => {
+
+  // available versions should be folders underneath this folder
+  let versions;
+  try {
+    versions = readdirSync(url, {withFileTypes: true}).filter(f=>f.isDirectory()).map(f=>f.name);
+    if (versions.length === 0) {
+      throw new Error('version list is empty');
+    }
+    versions = semverSort.desc(versions);
+  } catch(err) {
+    throw new Error(`Could not find versions of local package ${ name } at ${ url }. For more info about setting up a local feed, see here: https://docs.microsoft.com/en-us/nuget/hosting-packages/local-feeds`)
```

Comment:
nit: If surfaced to the UI, localize

Reply:
this gets localized in the ui via the standard error modal